components/multiselect/multiselect.jsx

```diff
@@ -85,7 +85,6 @@ export default class MultiSelect extends React.Component {
         } else {
             this.refs.list.setSelected(0);
         }
-        this.selected = null;
```

Comment:
I'm not quite sure what the issue causing the bug is yet but just doing this introduces a new bug:

1. Type in a few letters that start to match one user
2. Type in another character that causes no users to match
3. Hit enter

Observed: A user gets added
Expected: Nothing happens